scalableminds/webknossos
# #1714 Fix dataset Bounding Box

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Bug fix for dataset bounding box

Steps to test:
- Change `dataSource.dataLayers[*].sections[*].bboxBig.topLeft` in `datasource.json` & restart server
- Client should start the dataset bounding box from the new position.

Issues:
- fixes #1714

------
